data/json/snippets/nightmares.json

```diff
@@ -42,7 +42,12 @@
       "You dream that you're leaning over a bathroom sink, trying to pull loose a wisdom tooth that's been slicing up your gums.  You succeed in yanking it out and it clatters into the sink - the tooth is six inches long, jagged and serrated like the blade of a nasty dagger, with a crack in the enamel giving way to pink tissue underneath.  The taste of blood still lingers in your mouth for a moment even after you wake up.",
       "You have a long and vivid nightmare about living in an old, poorly ventilated building with countless other people.  The air is humid and stuffy, every room is brightly lit and the people are always yelling and making noise.  You try to get some rest but you can't.  You try using the restroom but it's too cramped.  It gets so unbearable that you feel sick, and you find that you've gagged yourself awake.",
       "You have a strange dream about walking through a street from your childhood, where every surface is the same shade of solid red.  You wake up startled after noticing an equally red man down the street, moving quickly in your direction.",
-      "In your dreams you feel as though you've learned something really, really important.  You wake up in quiet distress, as you've already forgotten what that something was."
+      "In your dreams you feel as though you've learned something really, really important.  You wake up in quiet distress, as you've already forgotten what that something was.",
+      "You open your eyes and find yourself in a school hallway.  It's high school.  Students are filing past one another to reach their respective classrooms.  You begin heading for your class as well, when you distinctly hear a soft voice crying out for help.  Startled, you go into a half-sprint and rush to where the noises are coming from.  You end up in front of one of the student bathrooms.  Although the weeping seems to be echoing from the ladies' restroom, you brace yourself and barge in.  The abrupt silence catches you off guard, and of the four stalls, one seems to be painted entirely black.  You creep towards it and silently push ajar the entrance.  You find with abject terror a naked baby blown to horrific proportions wailing out his lungs.  The macabre dread deadens your heart as you realize. This was how you were birthed into this world.",
```

Comment:
> Although the weeping seems to be echoing from the ladies' restroom, you brace yourself and barge in.

What if the player’s character is a lady themselves?

Reply:
Good point!